packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -226,7 +226,7 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
         if (this.state.outputStatus === ResponseStatus.COMPLETED && this.state.xyTree.length === 0 ) {
             return <React.Fragment>
                 {
-                    <p className='no-data' style={{fontSize: 20, marginRight: '5px', marginLeft: '5px', justifyContent:'center', alignItems:'center'}}>
```

Comment:
@bhufmann This is where the font size was originally set to 20. Now it's specified in the class instead. The change to the class should have been committed in this commit.

Reply:
Update: I ended up removing the size adjustment in the CSS class.